skunkworks_crow/src/main/java/org/odk/share/views/ui/settings/SettingsActivity.java

```diff
@@ -177,25 +183,70 @@ private ViewGroup getRootView() {
     }
 
     private void showPasswordDialog() {
-        AlertDialog.Builder builder = new AlertDialog.Builder(this);
         LayoutInflater factory = LayoutInflater.from(this);
 
         View dialogView = factory.inflate(R.layout.dialog_password_til, null);
+
         TextInputLayout tlPassword = dialogView.findViewById(R.id.et_password_layout);
         tlPassword.getEditText().setText(prefs.getString(PreferenceKeys.KEY_HOTSPOT_PASSWORD, getString(R.string.default_hotspot_password)));
 
+        edtpass = (TextInputEditText) dialogView.findViewById(R.id.edtpass);
```

Comment:
@devanshi7799 great work 🎉, but why are you making new callback of view `edtpass` till now we are  used `tlpassword.getEditText()` do not create two callbacks people may confuse between two make it one,

If you are thinking to use callback using `id`, do not use `findViewById` we are using one library called [Butter Knife](https://github.com/JakeWharton/butterknife) for callback of views can you please use that